_posts/2017-09-01-railsgirls-fairy-tail.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  Adventures in the Code Ocean – A RailsGirls Fairy Tail
```

Comment:
Is that spelling an intended pun? If not, please change it to 'Fairy Tale'.

Reply:
Oh, the typo is also in the branch name and I am not sure how to change it... But maybe it's okay because it wound be visible later?